src/components/ContentPreview/ContentPreview.js

```diff
@@ -713,7 +718,8 @@ class ContentPreview extends PureComponent<Props, State> {
             language,
             messages,
             className,
-            showSidebar,
+            contentSidebarProps,
+            contentSidebarProps: { showSidebar },
```

Comment:
I dont think this should be here

Reply:
eslint errors that i'm not using de-structuring if I don't include both

Reply:
I need to access both contentSidebarProps and contentSidebarProps.showSidebar

Reply:
👍 I assumed it was a typeo, but turns out this is how you can do nested destructuring. TIL.

IMO, ```const {showSidebar} = contentSidebarProps``` on the next line is less confusing, but am fine with either way

Reply:
yeah this looks weird, just put it on a new line